migrations/Version202307251344000003_taoAdvancedSearch.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
nitpick: Missing license header :see_no_evil:

Reply:
We are not using for any migration file. I am not sure it is needed, but I can add :)